### DISCUSSION TITLE
Running 'Python368.efi chipsec_main.py -m common.secureboot.variables -a modify' with Chipsec v1.9.1 failed with an exception.

### DISCUSSION THREAD

**Discussion by girishd22** [chipsec/chipsec#1625] [Q&A] [answered]
Hi Nathaniel Mitchell,
Running 'Python368.efi chipsec_main.py -m common.secureboot.variables -a modify' command in UEFI shell with Chipsec v1.9.1 failed with an exception "[-] ERROR: Exception occurred during chipsec.modules.common.secureboot.variables.run(): 'ord() expected string of length 1, but int found'"

Is there any file changes that needs to be done in order to get this to work?

Thanks,
Girish

**Comment by npmitche** [comment, marked as answer]
Can you try again with this fix: https://github.com/chipsec/chipsec/pull/1648

**Reply by npmitche** [reply]
Fix has been merged in. Please re-test with the top of main.
